mpas_analysis/shared/timekeeping/MpasRelativeDelta.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+import calendar
+
+
+class MpasRelativeDelta(relativedelta):
+    """
+    MpasRelativeDelta is a subclass of dateutil.relativedelta for relative time
+    intervals with different MPAS calendars.
+
+    Only relative intervals (years, months, etc.) are supported and not the
+    absolute date specifications (year, month, etc.).  Addition/subtraction
+    of datetime.datetime objects (but not other MpasRelativeDelta,
+    datetime.timedelta or other related objects) is supported.
```

Comment:
I don't think we have a use case where we need to be able to add or subtract two offsets.  I'm not sure if it's allowed for timedeltas.  I thought not but I'll check.